.teamcity/src/main/kotlin/configurations/DocsTest.kt

```diff
@@ -85,7 +85,7 @@ class DocsTestProject(
 }
 
 class DocsTestTrigger(model: CIBuildModel, docsTestProject: DocsTestProject) : BaseGradleBuildType(init = {
-    id("${docsTestProject.id}_Trigger")
+    id("${model.projectId}_DocsTest_${docsTestProject.testJava.version.name.toCapitalized()}_${docsTestProject.os.asName()}_Trigger")
```

Comment:
❓ I don't think we should use the Java version/OS in the ID of the `DocsTestTrigger,` similarly to what we do with the other test project. What is the problem when using `docsTestProject.id`?

Reply:
The generated id would be `Gradle_Master_Gradle_Master_Check_DocsTest_Java18_Macos_Trigger` instead of correct `Gradle_Master_Check_DocsTest_Java18_Macos_Trigger`.

Reply:
I think other ones just extract the generator into a function. We can do similar things here.